web/cobrands/fixmystreet/staff.js

```diff
@@ -248,6 +249,46 @@ fixmystreet.staff_set_up = {
         }
     }
 
+    function get_value_and_group(slr) {
+        var elt = $(slr)[0];
+        var group = $(elt.options[elt.selectedIndex]).closest('optgroup').prop('label');
+        return { 'value': $(elt).val(), 'group': group || '' };
+    }
+
+    function update_change_asset_button() {
+        var category = get_value_and_group('#category');
+        var found = false;
+        if (fixmystreet.assets) {
+            for (var i = 0; i < fixmystreet.assets.layers.length; i++) {
+                var asset_layer = fixmystreet.assets.layers[i];
+                if (asset_layer.relevant(category.value, category.group)) {
```

Comment:
This results in a crash when viewing the inspector report page if the layer doesn't have `asset_group` or `asset_category` (e.g. Lincolnshire's NSG layer).

Reply:
NSG layer shouldn't have been loaded because I skipped all VectorNearest layers? Hmm.

Reply:
Ah, LLPG layer, which is loaded as Asset because no usrn/road provided.

Reply:
Have pushed fix for that hopefully